data-dictionary.md

```diff
@@ -63,39 +60,255 @@ The RESO Data Dictionary consists of three main sets of data elements:
+| **LegacyODataValue**     | `Edm.String` | "ExclusiveAgency" | true | The Legacy OData lookup value that the server vendor provided in their OData XML Metadata.<br /><br />This value is optional, and has been included in order to provide a stable mechanism for translating OData lookup values to RESO standard lookup display names, as well as for historical data that might have included the OData value at some point, even after the vendor had converted to human friendly display names. |
+| **ModificationTimestamp** | `Edm.DateTimeOffset` | "2020-07-07T17:36:14+00:00" | **false** | The timestamp for when the enumeration value was last modified.<br /><br />This is used to help rebuild caches when metadata items change so consumers don't have to re-pull and reprocess the entire set of metadata when only a small number of changes have been made. |
+
+## Required Annotation
+For any String List, Single or String List, Multi field using the Lookup resource, the following MUST be present in the server metadata:
+
+### Example
+```xml
+<!-- OData annotation for String List, Single field -->
+<Property Name="OfficeCountyOrParish" Type="Edm.String">
+  <Annotation Term="RESO.OData.Metadata.LookupName" String="CountyOrParish" />  
+</Property>
+
+<!-- OData annotation for String List, Multi field -->
+<Property Name="ExteriorFeatures" Type="Collection(Edm.String)">
+  <Annotation Term="RESO.OData.Metadata.LookupName" String="ExteriorFeatures" />  
+</Property>
+```
+Where: 
+* `Term` uses the required namespace RESO.OData.Metadata.LookupName
+* `String` indicates the LookupName in the Lookup Resource in which the given field's lookups are defined. 
+
+Notes:
+* The referenced LookupName **MUST** be a standard lookup name for items currently defined by the RESO Data Dictionary. For example, for the `StandardStatus` field in the Data Dictionary, the `LookupName` **MUST** be `StandardStatus`.
+* Data providers **MAY** add additional LookupName entries when not already defined by the Dictionary.
+* The underlying type for the lookup-based field **MUST** either be `Edm.String` or `Collection(Edm.String)`, depending on whether the given field is String List, Single or Multi in the Data Dictionary, respectively.
+
+## Queries
+The `Lookup` resource **MUST** support queries that use the [OData`$top` and `$skip` query operators](https://docs.oasis-open.org/odata/odata/v4.01/odata-v4.01-part2-url-conventions.html#_Toc31361042), in conjunction with a `ModificationTimestamp` parameter so consumers can synchronize since the last update. The client **MUST** be able to consume the advertised count of records from the server or testing will not pass.
+
+Providers **MAY** support other queries on this resource, such as filtering by `LookupName`.
+
+### Example: GET Lookups using OData `$top` and `$skip` 
```

Comment:
If we go `nextLink` over `top` and `skip`  then this section will need to be updated.